package.json

```diff
@@ -35,15 +35,18 @@
     "lint": "eslint  \"packages/**/*.{js,ts}\" --fix",
     "compile": "cd packages/language-server && npm run build && cd ../vscode && npm run build && cd .. && cd ..",
     "watch": "npm run build -- -w",
-    "vsce:publish": "sh scripts/publish.sh",
-    "vsce:check": "npm run shell:lint && sh scripts/check-update.sh",
-    "vsce:bump": "sh scripts/bump.sh",
-    "shell:lint": "shellcheck -x scripts/**/*.sh",
+    "shell:lint": "shellcheck -x scripts/*.sh",
     "test": "sh scripts/e2e.sh",
     "testScripts": "jest __tests__",
-    "check:lsp": "sh scripts/lsp-version.sh",
-    "setup:patch": "sh scripts/patch/setup.sh",
-    "openvsx:publish": "cd packages/vscode && ovsx publish -p"
+    "openvsx:publish": "cd packages/vscode && ovsx publish -p",
+    "check_for_update": "sh scripts/check_for_update.sh",
+    "bump_prisma_version_in_scripts": "sh scripts/bump_prisma_version.sh",
+    "bump_extension_version_in_scripts": "node scripts/bump_extension_version.js",
```

Comment:
Is there a reason why the npm script and the actual script file differ in naming?

Reply:
no, I replaced those with javascript files now and called them directly